docs/administration/password-recovery.md

```diff
@@ -0,0 +1,16 @@
+# Password recovery by user
+
+If user has forgotten his password and wants to recover it on his own, he needs to do the following:
+
+1. Go to the login page and click **Forgot Password?**.
+2. Enter login and email, then click **Submit**.
+
+After that, user will receive an email with a link, which he must follow to enter a new password and confirm it.
+
+---
+
+**Important:** It is advisable not to delete the email sent to you with a link to reset your password until you have finally changed your password. Because EspoCRM has a setting that prevents new requests from being created (1 request per 3 hours) until you follow the link in the first password recovery request and complete this operation.
+
+For example, you requested password recovery and did not find the email or accidentally deleted it. In this case, you will need to wait 3 hours before you can request a new password reset.
```

Comment:
> It is advisable not to delete the email sent
I believe In like 95% emails are not deleted but in trash folder. Not sure whether we need to emphasize about this possible case, at lease it's not to mark is as important to not to distract the reader.